py/sentry_streams/example_config.py

```diff
@@ -9,9 +9,16 @@
     logical_topic="logical-events",
 )
 
+map = Map(
+    name="mymap",
+    ctx=pipeline,
+    inputs=[source],
+    function="sentry_streams.sample_function.EventsPipelineMapFunction.simple_map",
```

Comment:
Is it intentional to make this a bare string and not support a reference to a function ? 
Supporting references to a Callable:
Pros:
- type checking that the path is a valid function
Cons:
- That function has to be imported in this file which encourages people to write the function here.

Reply:
Left a comment to get back to this